_core/delete/delete.php

```diff
@@ -69,16 +69,16 @@ private function delete_post($no, $pwd, $imgonly = 0, $automatic = 0, $children
             //$mysql->query("INSERT INTO " . SQLDELLOG . " (admin, type, action, time, board, postno) VALUES ('{$auser}', '2', 'Deleted post #{$no}', '" . time() . "', '" . BOARD_DIR . "', '{$no}')");
         }
         
-        $conditions = ($row['resto'] == 0 && $children && !$imgonly) ? "(no='{$no}' OR resto='{$no}')" : "no='{$no}'"; 
-        $result = $mysql->query("SELECT * FROM " . SQLLOG . " WHERE {$conditions} AND board='" . BOARD_DIR . "'");
+        $conditions = ($row['resto'] == 0 && $children && !$imgonly) ? "(no=:no OR resto=:no)" : "(no=:no)";
```

Comment:
I'm not sure, just seems longer than it needs to be. We're checking three conditions just to enable one.